plasmapy/formulary/parameters.py

```diff
@@ -528,35 +535,54 @@ def ion_sound_speed(
 cs_ = ion_sound_speed
 """ Alias to :func:`ion_sound_speed`. """
 
+# -----                                                             thermal_speed  -----
 
 # This dictionary defines coefficients for thermal speeds
 # calculated for different methods and values of ndim.
 # Created here to avoid re-instantiating on each call
-_coefficients = {
+#
+thermal_speed_coefficients = {
     1: {"most_probable": 0, "rms": 1, "mean_magnitude": 2 / np.pi},
     2: {"most_probable": 1, "rms": 2, "mean_magnitude": np.pi / 2},
     3: {"most_probable": 2, "rms": 3, "mean_magnitude": 8 / np.pi},
 }
+"""
+Dictionary of various coefficients used in calculating the
+`~plasmapy.formulary.parameters.thermal_speed`.
+"""
+
+
+@preserve_signature
+@njit
+def thermal_speed_lite(T, mass, coeff):
+    """
```

Comment:
This docstring should still provide information on the parameters and returns so those appear in the documentation. Right now in the docs there's no indication that this function doesn't take units, or where one should look to determine `coeff`